css/index.css

```
@@ -773,4 +772,215 @@ body {
 
 .food-spin__location-indicator{
   stroke: var(--title-color);
-}
\ No newline at end of file
+}
+
+.cart-summary {
+  max-width: 500px;
+  background-color: var(--color-white);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  position: fixed;
+  top: 0%;
+  right: 0;
+  z-index: 1000;
+  height: 100%;
+  border: none;
+  padding: 0px;
+  --text-color: #FF922C;
+  --button-color: #FF6600;
+  --box-shopping: rgba(255, 102, 0, 0.25);
+  box-shadow: 0px 0px 70px 20px var(--box-shopping);
+  transform: translateX(200%);
+  transition: transform 0.4s ease;
+  opacity: 0;
+}
+
+.cart-summary[open] {
+  transform: translateX(0);
+  opacity: 1;
+}
+
+.cart-summary__close {
+  color: var(--text-color);
+  font-size: 24px;
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+.cart-summary__top {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px;
+  border-bottom: 3px solid #F5F5F5;
+}
+
+.cart-summary__title {
+  color: var(--text-color);
+  font-size: 28px; 
+  font-weight: var(--font-weight-boldX);
+  line-height: 24px;
+  letter-spacing: 0.02em;
+}
+
+.cart-summary__svg{
+  fill: var(--text-color);
+}
+
+.cart-summary__content {
+  padding: 30px;
+}
+
+.cart-summary__item{
+  margin-bottom: 15px;
+}
+
+.cart-summary__description {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+}
+
+.cart-summary__name,
+.cart-summary__prics{
+  font-weight: var(--font-weight-boldX);
+  font-size: 24px;
+  line-height: 32px;
+  color: var(--text-color);
+}
+
+.cart-summary__discount {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+  margin-top: 15px;
+}
+
+.cart-summary__discount-description,
+.cart-summary__discount-price{
+  font-weight: var(--font-weight-boldX);
+  font-size: var(--sixteen-pixeles);
+  line-height: var(--eighteen-pixel);
+  color: var(--color-black-medium);
+}
+
+.cart-summary__discount-price__remove {
```

Error de principiante 😄 . En BEM no se usa mas de una vez `__`
```suggestion
.cart-summary__discount-price-remove {
```